# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1758806400 -7200
# Parent  b64ed2173d92fe55c9027511d91ddf68f712c355
Misc: Switch off new Account Hub (too buggy) and fix Windows font rendering (bug 1946024).

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1485,9 +1485,10 @@ pref("mail.thread.conversation.enabled",
 pref("mail.dark-reader.enabled", true);
 pref("mail.dark-reader.show-toggle", true);
 
 // Enable the new account setup (starting from the second account)
-pref("mail.accounthub.enabled", true);
+// This has too many errors.
+pref("mail.accounthub.enabled", false);
 
 // Enable address book setup via account hub
 pref("mail.accounthub.addressbook.enabled", false);
 
@@ -1504,4 +1505,9 @@ pref("calendar.dialogs.new.enabled", fal
 pref("mail.threadpane.listview", 0);
 
 // Row count for the cards view, currently bound to a range between 2 and 3.
 pref("mail.threadpane.cardsview.rowcount", 3);
+
+// Fit font rendering.
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_max_size", 15);
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_for_families",
+  "Arial,Consolas,Courier New,Microsoft Sans Serif,Segoe UI,Tahoma,Trebuchet MS,Verdana");
